gui/build.py

```diff
@@ -80,14 +80,21 @@ def do_bundle(options):
         if os.path.isdir(dirname):
             shutil.rmtree(dirname)
     # create a intaller
+    if IS_WIN:
```

Comment:
We don't need the IS_WIN check yet - for now this part of the script is windows only (we will make it work for linux with the next bug). So please remove it for now. Also, the default values for python_path and nsis_path can be defined as a default= argument in the add_argument call, so you can just remove all this block.